src/Parser/FrostMage/Modules/Features/ThermalVoidDuration.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import Module from 'Parser/Core/Module';
+
+import { formatNumber } from 'common/format';
+
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+
+class ThermalVoidDuration extends Module {
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasTalent(SPELLS.THERMAL_VOID_TALENT.id);
+  }
+
+  suggestions(when) {
```

Comment:
This whole suggestion is copied from Shadow Priest and wasn't changed.

Reply:
Oops, ill fix that.